### PR TITLE
63 style and create favourites page and discover page copy and paste with different route

### PR DESCRIPTION
**Pull request by GeorgeSwift16**
Tested, main merged.

### CODE REVIEW

File: src/components/Card/__snapshots__/Card.test.jsx.snap [hunk 1]
```diff
@@ -3,7 +3,7 @@
 exports[`should render the text on the display 1`] = `
 <div>
   <div
-    class="card"
+    class="card card--undefined"
```
Comment (DanForder) [inline src/components/Card/__snapshots__/Card.test.jsx.snap:6]: update code so these don't come through as undefined.. either update the tests to have a prop filled, or update the component so there's a default value

File: src/components/TextField/__snapshots__/TextField.test.jsx.snap [hunk 2]
```diff
@@ -3,10 +3,10 @@
 exports[`should render the TextField component 1`] = `
 <div>
   <div
-    class="text-field"
+    class="text-field text-field--undefined"
```
Comment (DanForder) [inline src/components/TextField/__snapshots__/TextField.test.jsx.snap:6]: same as above here, make sure this isn't undefined

File: src/pages/Contact/__snapshots__/Contact.test.jsx.snap [hunk 3]
```diff
@@ -62,10 +62,10 @@ exports[`should render contact page 1`] = `
         class="contact__form"
       >
         <div
-          class="text-field"
+          class="text-field text-field--undefined"
```
Comment (DanForder) [inline src/pages/Contact/__snapshots__/Contact.test.jsx.snap:65]: same as above, make sure this isn't undefined

File: src/pages/Home/Home.test.jsx [hunk 4]
```diff
@@ -15,3 +17,11 @@ it("should render the text on the display", () => {
 
   expect(container).toMatchSnapshot();
 });
+
+it("should render the list view page when a user clicks list view", () => {
+  customRender(<Home />);
+  const { container } = customRender(<HomeListView />);
+  const listViewLink = screen.getByText("List View");
+  userEvent.click(listViewLink);
+  expect(container).toBeInTheDocument();
```
Comment (DanForder) [inline src/pages/Home/Home.test.jsx:26]: isn't `container` always going to be in the document? might need a different way of testing this..

File: src/pages/HomeListView/HomeListView.test.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+  expect(container).toMatchSnapshot();
+});
+
+it("should render the home page when a user clicks map-view", () => {
+  customRender(<HomeListView />);
+  const { container } = customRender(<Home />);
+  const mapViewLink = screen.getByText("Map View");
+  userEvent.click(mapViewLink);
+  expect(container).toBeInTheDocument();
```
Comment (DanForder) [inline src/pages/HomeListView/HomeListView.test.jsx:18]: same as above, does this actually test the URL change?

**Review by DanForder — CHANGES_REQUESTED** [review]

**Review by DanForder — APPROVED** [review]

**Review by ekauf — APPROVED** [review]

**Pull request merged**